packages/workspace/src/generators/move/lib/move-project-configuration.ts

```diff
@@ -12,6 +13,7 @@ export function moveProjectConfiguration(
   schema: NormalizedSchema,
   projectConfig: ProjectConfiguration & NxJsonProjectConfiguration
 ) {
+  const isStandalone = isStandaloneProject(tree, schema.projectName);
```

Comment:
Can we use `getWorkspaceLayout(host).standaloneAsDefault` so we don't have to add the new function?

It should work in this case because the current move logic already moves the `project.json` file appropriately.

From what I saw, this could also be solved somewhat hackily by moving `addProjectConfiguration` after `removeProjectConfiguration`. But I think reading whether it should be standalone or not and then explicitly saying to add a standalone is a better option.

Reply:
That wouldn't be right for a couple of reasons:

- `getWorkspaceLayout(host).standaloneAsDefault` will only be `true` if all projects in the workspace are configured as standalone, you'd expect that to normally be the case, either all or none, but that's an assumption, not a fact. Even our generator to convert projects into standalone config offers the ability to specify a project, it's not mandatory to convert the whole workspace.
- It can't be solved by having `addProjectConfiguration` before `removeProjectConfiguration`, because there are cases where the project name for the destination is the same as the old destination (e.g. moving `domain-project` to `domain/project`, both results in the name `domain-project`). A couple of issues have been created because of that, and I previously worked on solving those. In that scenario, if you try to add the project first, you get an error because it already exists.

The need here is to know if the project being moved is configured as standalone or not, that doesn't match what `getWorkspaceLayout(host).standaloneAsDefault` returns (whether all projects in the workspace are configured as standalone).

Reply:
Ah okay, I see